dotCMS/src/main/java/com/dotcms/rest/elasticsearch/ESContentResourcePortlet.java

```diff
@@ -40,14 +43,22 @@ public Response search(@Context HttpServletRequest request, @Context HttpServlet
 		User user = initData.getUser();
 		ResourceResponse responseResource = new ResourceResponse(initData.getParamsMap());
 
+		if(LicenseUtil.getLevel() < LicenseLevel.STANDARD.level){
+			final String noLicenseMessage = "Unable to execute ES API Requests. Please apply an Enterprise License";
+			final RuntimeException e1 = new RuntimeException(noLicenseMessage);
+			Logger.warn(this.getClass(), noLicenseMessage);
+			return ExceptionMapperUtil.createResponse(e1, Response.Status.BAD_REQUEST);
```

Comment:
Man you can do something like:

```
Response
                .status(status)
                .entity(map("message", noLicenseMessage))
                .header("error-message", noLicenseMessage)
                .build();
```

Reply:
ok then, I'll update it first thing in the morning.